### PR TITLE
[T5A3][W13-B1] Song Zhiwen

### PR DESCRIPTION
**Pull request by zzzzwen**

### CODE REVIEW

File: src/seedu/addressbook/ui/DarkTheme.css [hunk 1]
```diff
@@ -1,17 +1,19 @@
 
 
 .text-field {
+    -fx-background-color: white;
     -fx-font-size: 12pt;
-    -fx-font-family: "Consolas";
+    -fx-font-family: "Arial";
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/DarkTheme.css]: try different things/settings to get a hang of things in JavaFx!

File: src/seedu/addressbook/ui/Gui.java [hunk 2]
```diff
@@ -34,10 +35,11 @@ public void start(Stage stage, Stoppable mainApp) throws IOException {
         mainWindow.displayWelcomeMessage(version, logic.getStorageFilePath());
     }
 
-    private MainWindow createMainWindow(Stage stage, Stoppable mainApp) throws IOException{
+    private MainWindow createMainWindow(Stage stage, Stoppable mainApp) throws IOException {
         FXMLLoader loader = new FXMLLoader();
         loader.setLocation(Main.class.getResource("ui" + File.separator + "mainwindow.fxml"));
         stage.setTitle(version);
+        stage.getIcons().add(new Image(Gui.class.getResourceAsStream("addressbook.png")));
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Gui.java]: don't you need to give the full path here?

**Comment by okkhoy** [comment]
@zzzzwen 
Some comments added. Please ack & close the PR after reading comments.

**Comment by zzzzwen** [comment]
noted

**Pull request closed without merging**